pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
hold on... I'm trying to understand what we're attempting to enable here, what old clients we're concerned about, and what we want the behavior to be:

* old clients will always sent `netid:<number>`
* old clients will read a missing `netid` as `0`, and will probably choke on a `netid:null` sent through the API
* if we leave the external type `uint`, new clients will behave the same way
* if we make the external type *uint, new clients have the potential to send `netid:null`, and receive `netid:null`

I would like to make the external type *uint, but the following questions need answering:
* what old clients exist for this API?
* what are the implications if one of them misinterprets a missing netid as 0?